opentelemetry-otlp/examples/basic-otlp-http/src/main.rs

```diff
@@ -74,8 +74,8 @@ static COMMON_ATTRIBUTES: Lazy<[KeyValue; 4]> = Lazy::new(|| {
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
-    let _ = init_tracer()?;
-    let _ = init_metrics()?;
+    init_tracer()?;
```

Comment:
If the error was reported for unused return, can we instead handle the return value. Something like (not tested):
```rust
        let result = init_tracer();
        assert!(result.is_ok(), "Init tracer failed with error: {:?}", result.err());
```